src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/JavaNCSSCheck.java

```diff
@@ -453,6 +495,18 @@ private static boolean isExpressionCountable(DetailAST ast) {
         return countable;
     }
 
+    /**
+     * Checks if a token is a method, constructor, or compact constructor definition.
+     *
+     * @param tokenType the type of token we are checking
+     * @return true if token type is method or ctor definition, false otherwise
+     */
+    private static boolean isMethodOrCtorDefinition(int tokenType) {
```

Comment:
I needed to create this to reduce cyclomatic complexity in the `leaveToken()` method, so I also used this method in the` visitToken()` method.

Reply:
I thought about that, but honestly I couldn't think of a good name for the method.

Reply:
Fixed.

Reply:
Fixed.